components/EntriesList/EntriesList.js

```diff
@@ -46,9 +46,9 @@ export default function EntriesList() {
               exit={{ opacity: 0, scale: 0.8, transition: { duration: 0.3 } }}
             >
               <Styled.Card onClick={() => router.push(`${entry._id}`)}>
-                <Styled.AnimationContainer>
+                <Styled.ColoredShapeContainer>
```

Comment:
Do you need to call your styled components Styled.xyz because of how you you import them or could you also omit the "Styled." if so, I'd do it for readability

Reply:
yes, its imported like this: import * as Styled from "./EntriesList.styled";